scripts/gallery.gd

```diff
@@ -18,6 +18,288 @@ func create(code):
 	else:
 		return "Character not found: " + code
 
+var characters = {
```

Comment:
This entire file is a regression and should be deleted.

Reply:
Just reverting it